### DISCUSSION TITLE
Does SkyWalking Satellite support for Zabbix agent

### DISCUSSION THREAD

**Discussion by naveenkesari** [apache/skywalking#12398] [Q&A] [answered]
Hi Team,

Does SkyWalking Satellite support for Zabbix agent to collect the metrics data and send back to oap server.

Thanks,
Naveen

**Comment by wu-sheng** [comment, marked as answer]
Satellite doesn't collect metrics from any place. It is just a proxy and load balancer for skywalking native protocols.

**Reply by naveenkesari** [reply]
Thanks for the update, it means on Zabbix agent side we can give the Satellite or LB IP and then it will send back the data to OAP server, please confirm if i am correct.

**Reply by wu-sheng** [reply]
No, we don't support that for now. As Zabbix protocol is not supported by Satellite.

**Reply by wu-sheng** [reply]
If you want to contribute that feature, we could help on reviewing it.
